ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Nti/AddConditions.cshtml.cs

```diff
@@ -0,0 +1,173 @@
+﻿using ConcernsCaseWork.Mappers;
+using ConcernsCaseWork.Models;
+using ConcernsCaseWork.Models.CaseActions;
+using ConcernsCaseWork.Pages.Base;
+using ConcernsCaseWork.Services.Nti;
+using ConcernsCaseWork.Services.NtiWarningLetter;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Service.Redis.Nti;
+using Service.Redis.NtiWarningLetter;
+using Service.TRAMS.Nti;
+using Service.TRAMS.NtiWarningLetter;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.Pages.Case.Management.Action.Nti
+{
+	[Authorize]
+	[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+	public class AddConditionsPageModel : AbstractPageModel
+	{
+		private readonly INtiWarningLetterStatusesCachedService _ntiWarningLetterStatusesCachedService;
+		private readonly INtiWarningLetterReasonsCachedService _ntiWarningLetterReasonsCachedService;
+		private readonly INtiModelService _ntiModelService;
+		private readonly INtiConditionsCachedService _ntiConditionsCachedService;
+		private readonly ILogger _logger;
+
+		[TempData]
+		public string ContinuationId { get; set; }
+
+		[TempData]
+		public bool IsReturningFromConditions { get; set; }
+
+		public long CaseUrn { get; private set; }
+		public long? NtiId { get; set; }
+
+		public ICollection<NtiConditionModel> SelectedConditions { get; private set; }
+		public ICollection<NtiConditionDto> AllConditions { get; private set; }
+
+		public AddConditionsPageModel(INtiWarningLetterStatusesCachedService ntiWarningLetterStatusesCachedService,
+			INtiWarningLetterReasonsCachedService ntiWarningLetterReasonsCachedService,
+			INtiModelService ntiModelService,
+			INtiConditionsCachedService ntiConditionsCachedService,
+			ILogger<NtiWarningLetterConditionModel> logger)
```

Comment:
Wrong class in logger, is NtiWarningLetterConditionModel, should be AddConditionsPageModel